pkg/storage/tsdb/config.go

```diff
@@ -292,6 +292,23 @@ type BucketStoreConfig struct {
 
 	// Controls how many series to fetch per batch in Store Gateway. Default value is 10000.
 	SeriesBatchSize int `yaml:"series_batch_size"`
+
+	// Token bucket configs
+	TokenBucketLimiter TokenBucketLimiterConfig `yaml:"token_bucket_limiter"`
+}
+
+type TokenBucketLimiterConfig struct {
+	Enabled                    bool    `yaml:"enabled"`
+	DryRun                     bool    `yaml:"dry_run"`
+	PodTokenBucketSize         int64   `yaml:"pod_token_bucket_size"`
```

Comment:
Just a nit. What about renaming `Pod` to `Instance` or something non-K8s related?

Reply:
Yup will do